docs: change global configuration order wording (#15866)
vuetifyjs/vuetify: packages/docs/src/pages/en/features/global-configuration.md

When creating and mounting a component, Vuetify uses the following priority in determining which default prop value to use:

> 1. Value defined in global section of `defaults` configuration object
> 2. Value defined in component specific section of `defaults` configuration object
> 3. Value defined in the component definition

Author: ```suggestion
1. Value set as prop value to the component itself
2. Value defined in component specific section of defaults configuration object
3. Value defined in global section of defaults configuration object
4. Value defined in the component definition by Vuetify
```

> When creating and mounting a component, Vuetify uses the following priority in determining which default prop value to use:

Author: ```suggestion
When creating and mounting a component, Vuetify uses the following priority in determining which prop value to use:
```

> 1. Value set as prop value to the component itself
> 2. Value defined in component specific section of defaults configuration object
> 3. Value defined in global section of defaults configuration object
> 4. Value defined in the component definition by Vuetify

Author: ```suggestion
1. Value set as prop value to the component itself
2. Value defined in component specific section of defaults configuration object
3. Value defined in global section of defaults configuration object
4. Value defined in the prop definition of the Vueitfy component itself.
```
